bin/mail/2016-08-09-hoc-latam/generate-mailing-lists.rb

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# Latin America (latam), 2 sub-lists for es & pt
+
+LATAM_ES_COUNTRIES = [
+  'Mexico',
+  'Guatemala',
+  'El Salvador',
+  'Costa Rica',
+  'Nicaragua',
+  'Belize',
+  'Panama',
+  'Cuba',
+  'Dominican Republic',
+  'Venezuela',
+  'Colombia',
+  'Ecuador',
+  'Peru',
+  'Bolivia',
+  'Paraguay',
+  'Uruguay',
+  'Chile',
+  'Argentina',
+  'Puerto Rico'
+].freeze
+es_country_or_list = LATAM_ES_COUNTRIES.join(' OR ')
+
+LATAM_ES_COUNTRY_CODES = %w(mx gt sv cr ni bz pa cu do ve co ec pe bo py uy cl ar pr).freeze
+es_country_code_or_list = LATAM_ES_COUNTRY_CODES.join(' OR ')
+
+def query_from_list(queries)
+  {}.tap do |contacts|
+    # Reverse sort to start with more recent years.
+    # query_subscribed_contacts dedupes by rejecting duplicate emails.
+    queries.keys.sort.reverse_each do |query_name|
```

Comment:
No need to change (in fact, please don't)... But this seems awfully convoluted to accomplish what you are accomplishing. Why not define the hash in reverse order and iterate over `|query_name, query|` (rather than sorting, reversing, and grabbing the value from the key)?